python-discord/site
# Dewikify!

This PR removes wiki features and django-allauth authentication from the site.

- Event pages and sidebars are static pages at `pydis_site/templates/events/pages`
- Resources are now loaded from `.yaml` files at `pydis_site/apps/resources/resources`
- Content pages (guides, rules, etc.) are now loaded from markdown at `pydis_site/apps/content/resources/`

### How to Review
1. Run the branch locally.
2. Compare each page with the corresponding page on the current website.
3. Don't suggest grammar/phrasing changes for any existing page content.
4. Check if links function as intended.
5. Additionally, check if site links redirect to their dewikification counterpart on the local site. For example, `/pages/roles/` should redirect to `/pages/server-info/roles/`.
